src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1

```diff
@@ -68,66 +68,66 @@ this parameter.
+and a moderator (if applicable), plus any moderation review has 
+intended for you to receive with this package. If you are looking at a 
+remote source that uses the same url for updates, you will need to 
+ensure the package also stays updated in line with those remote 
+resource updates. You should look into [automatic packaging](https://chocolatey.org/docs/automatic-packages) 
+to help provide that functionality.
+
+.PARAMETER ChecksumType
+The type of checkum that the file is validated with - valid
+values are 'md5', 'sha1', 'sha256' or 'sha512' - defaults to 'md5'.
+
+MD5 is not recommended as certain organizations need to use FIPS
+compliant algorithms for hashing - see
+https://support.microsoft.com/en-us/kb/811833 for more details.
+
+The recommendation is to use at least SHA256.
+
+.PARAMETER Checksum64
+OPTIONAL if no Url64bit - The checksum hash value of the Url64bit
+resource. This allows a checksum to be validated for files that are not
+local. The checksum type is covered by ChecksumType64.
+
+**NOTE:** Checksums in packages are meant as a measure to validate the 
+originally intended file that was used in the creation of a package is
+the same file that is received at a future date. Since this is used for
+other steps in the process related to the community repository, it 
+ensures that the file a user receives is the same file a maintainer
+and a moderator (if applicable), plus any moderation review has 
+intended for you to receive with this package. If you are looking at a 
+remote source that uses the same url for updates, you will need to 
+ensure the package also stays updated in line with those remote 
+resource updates. You should look into [automatic packaging](https://chocolatey.org/docs/automatic-packages) 
 to help provide that functionality.
 
 **NOTE:** To determine checksums, you can get that from the original 
 site if provided. You can also use the [checksum tool available on 
 the community feed](https://chocolatey.org/packages/checksum) (`choco install checksum`) 
 and use it e.g. `checksum -t sha256 -f path\to\file`. Ensure you 
-provide checksums for all remote resources used.
-
-.PARAMETER ChecksumType64
-OPTIONAL - The type of checkum that the file is validated with - valid
-values are 'md5', 'sha1', 'sha256' or 'sha512' - defaults to
-ChecksumType parameter value.
-
-MD5 is not recommended as certain organizations need to use FIPS
-compliant algorithms for hashing - see
+provide checksums for all remote resources used.
+
+.PARAMETER ChecksumType64
+OPTIONAL - The type of checkum that the file is validated with - valid
+values are 'md5', 'sha1', 'sha256' or 'sha512' - defaults to
+ChecksumType parameter value.
+
+MD5 is not recommended as certain organizations need to use FIPS
+compliant algorithms for hashing - see
 https://support.microsoft.com/en-us/kb/811833 for more details.
-
-The recommendation is to use at least SHA256.
+
+The recommendation is to use at least SHA256.

```

Comment:
I'm betting it was Vim and/or Git messing with the line endings, nothing seems to be missing just "changed".